### PR TITLE
Seen Posts: Hide seen posts completely

### PR DESCRIPTION
**Pull request by char-lock**
### Description

As per #869, adds a preference to the Seen Posts script to allow users to hide seen posts on the dashboard using a CSS style.

### Testing steps

User should be able to browse as usual, but when their dashboard updates with new posts, any posts they have viewed will be hidden, leaving only new posts and old ones that have not been viewed.

### CODE REVIEW

File: src/scripts/seen_posts.css [hunk 1]
```diff
@@ -2,3 +2,7 @@ body:not(.xkit-seen-posts-only-dim-avatar) .xkit-seen-posts-seen:not(:hover),
 body.xkit-seen-posts-only-dim-avatar .xkit-seen-posts-seen:not(:hover) article > :first-child img {
   opacity: 0.5;
 }
+
+body.xkit-seen-posts-hide .xkit-seen-posts-seen {
+  display: none;
+}
```
Comment (marcustyphoon) [inline src/scripts/seen_posts.css:8]:
```suggestion
body.xkit-seen-posts-hide .xkit-seen-posts-seen article {
  display: none;
}
```

Hiding only the article element ensures that k-to-scroll-up still works; without this, if the post above the one you're pressing k while looking at is hidden with this extension, Tumblr's code will be unable to determine its location and will scroll to the top of the page.

File: src/scripts/seen_posts.css [hunk 2]
```diff
@@ -3,6 +3,6 @@ body.xkit-seen-posts-only-dim-avatar .xkit-seen-posts-seen:not(:hover) article >
   opacity: 0.5;
 }
 
-body.xkit-seen-posts-hide .xkit-seen-posts-seen article {
+body.xkit-seen-posts-hide .xkit-seen-posts-seen {
```
Comment (marcustyphoon) [inline src/scripts/seen_posts.css:6]: Not sure why you reverted this!

File: src/manifest.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
   "manifest_version": 2,
   "name": "XKit Rewritten",
-  "version": "0.21.0",
+  "version": "0.21.1",
```
Comment (marcustyphoon) [inline src/manifest.json:4]: We don't do version bumps of the extension along with script updates!

Comment (char-lock) [reply src/manifest.json:4]: That was an accident! Sorry about that.

Comment (char-lock) [reply src/manifest.json:4]: I don't even remember why I did that/pushed it. Ough.

**Comment by char-lock** [comment]
It's been long enough on this PR that I don't much remember the work I did on it. Should I just go ahead and close it and reopen whenever I go through and fix the issues, just to save the headache?

**Comment by AprilSylph** [comment]
@char-lock up to you! Sorry that we (I) let this sit so long.

**Comment by marcustyphoon** [comment]
Yep, either way. I could take it over as well; it's similar enough to Show Originals.

**Comment by char-lock** [comment]
> Yep, either way. I could take it over as well; it's similar enough to Show Originals.

If you want to take it over, please feel free!

I appreciate y'all's work on the plugin and administrative handling, by the way!

**Pull request closed without merging**